app/qml/LayerAttributesPanel.qml

```diff
@@ -0,0 +1,32 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.12
+
+Item {
+  id: root
+
+  property string selectedLayer: ""
+  signal backButtonTapped()
```

Comment:
backButtonClicked?

Reply:
https://uiux.blog/stop-using-the-term-click-in-a-mobile-experience-479b8ed4f567

Reply:
well, in that case we would need to rewrite all other signal so it uses the same 

also we use qt signals that uses "click"

Reply:
Your call guys :)

Reply:
Go with Peter's suggestion! I just send random comments and you can ignore tham